pallets/ddc-staking/src/lib.rs

```diff
@@ -173,14 +173,14 @@ pub mod pallet {
 	pub type Ledger<T: Config> =
 		StorageMap<_, Blake2_128Concat, T::AccountId, StakingLedger<T::AccountId, BalanceOf<T>, T>>;
 
-	/// The map of (wannabe) CDN participants stash keys to the DDC cluster ID they wish to
+	/// The map of (wannable) CDN participants stash keys to the DDC cluster ID they wish to
```

Comment:
The comment from the FRAME stacking pallet, could be rephrased.

Reply:
Agree, I didn't know there is "wannabe" word in English, so I thought it's a typo 😅